examples/vitess/vtctld-pod.yaml

```diff
@@ -0,0 +1,42 @@
+kind: Pod
```

Comment:
You don't need this to keep running?

https://github.com/kubernetes/kubernetes/blob/master/docs/user-guide/pods.md#durability-of-pods-or-lack-thereof